CONTRIBUTING.md

```diff
@@ -86,6 +86,17 @@ Makefile allow you to install needed tools:
 $ make install.tools
 ```
 
+### Prerequisite before build
+
+You need install some dependencies before building a binary..
+
+#### Fedora
+
+  ```shell
+  $ sudo dnf install gpgme-devel libseccomp-devel.x86_64 libseccomp-devel.x86_64
```

Comment:
I think we should recommend systemd-devel as well (it's optional but strongly recommended).

Reply:
Don't we already have a big section on this in building section on podman.io?

Reply:
@mheon I've added systemd-devel. 
@rhatdan I can't find building section on podman.io..

Reply:
This is the link: https://podman.io/getting-started/installation#build-and-run-dependencies
I think adding the link should be sufficient.